kernel/src/main/scala/me/rexim/morganey/syntax/LambdaParser.scala

```diff
@@ -14,7 +14,9 @@ object IntMatcher {
     }
 }
 
-object LambdaParser extends RegexParsers {
+object LambdaParser extends LambdaParser
```

Comment:
Inheritance from your parser was necessary for code reuse (since the interpolator also requires a parser, but with the capability for parsing special names for holes used in interpolation).

The companion object is used by your code in the Main-class. To not break your repl I kept the api which was exposed by your parser.